cirras-underwriting-war/cirras-underwriting-war/src/main/angular/src/app/components/inventory/forage/forage-inventory.component.html

```diff
@@ -285,7 +285,7 @@ <h2>Forage Field Inventory</h2>
                                                             matTooltip="Please enter seeding year" 
                                                             (keypress)="numberOnly($event)" 
                                                             (keydown)="navigateUpDown($event, 3)"
-                                                            (blur)="isMyFormDirty()"
+                                                            (blur)="clearPlantInsurability(fieldIndex, plantingIndex, invSeededIndex)"
```

Comment:
Shouldn't clearPlantInsurability be called in seedingDate onBlur as well?

Reply:
No, because seedingDate  is visible only for Silage Corn, and when Silage corn is selected then the plant insurability is not visible.

Reply:
In that case, why does getVarietySddYear include seedingDate and not just seedingYear?